Source/Core/Core/IOS/Device.cpp

```diff
@@ -76,6 +76,15 @@ bool IOCtlVRequest::HasInputVectorWithAddress(const u32 vector_address) const
                      [&](const auto& in_vector) { return in_vector.address == vector_address; });
 }
 
+bool IOCtlVRequest::HasNumberOfVectors(const size_t in_count, const size_t io_count) const
+{
+  if (in_vectors.size() != in_count || io_vectors.size() != io_count)
+    return false;
```

Comment:
Should probably put a newline after this to visually separate the check from the actual behavior.